### PR TITLE
feat: add bug bounty program

### PR DESCRIPTION
**Pull request by johnrwatson**
Adds bug bounty to System Initiative's Production Stack

### CODE REVIEW

File: BUG BOUNTY [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+The associated rules, qualification and rewards for this program may be revised at any time and any decisions made by security@systeminit.com are considered final.
+
+# Contained within this document:
+- **Disclosure Guidelines**: The rules and guidelines for reporting vulnerabilities.
+- **Rewards**: Information on bounty payouts based on severity.
+- **Ineligible Findings**: Specific exclusions from the policy.
+- **Scope**: Details about what assets are in scope and out of scope.
+
+System Initiative's Bounty Standards set expectations for both System Initiative and hackers when assessing the reward for a report. The standards begin with a relevant rating system such as CVSS for report assessments and then additionally consider additional adjustments ("bumps") for discrepancies between associated rating scores and actual business impact. This ensures more accurate correlations between vulnerability impact evaluations to reward bounties.
```
Comment (keeb) [inline BUG BOUNTY:19]: Could link to CVSS ratings here.

**Review by keeb — COMMENTED** [review]
This looks great to me, added a minor suggestion. Nice!

**Review by britmyerss — APPROVED** [review]
Great work!!

File: BUG BOUNTY.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
+# Bug Bounty at System Initiative
+
+**Launch Date:** 27th November, 2024 | **Last updated:** 27th November, 2024
```
Comment (nickgerace) [inline BUG BOUNTY.md:3]: Is this not implied by the git history? If we want this, that works for me! Just curious.

Comment (johnrwatson) [reply BUG BOUNTY.md:3]: In theory yes, might be worth removing the Last Updated. Launch date is still relevant

Comment (johnrwatson) [reply BUG BOUNTY.md:3]: Sorry i missed this before merging, I can come back to it and remove it

**Pull request merged**